types/node/crypto.d.ts

```diff
@@ -654,6 +655,9 @@ declare module 'crypto' {
     interface CipherGCMOptions extends stream.TransformOptions {
         authTagLength?: number | undefined;
     }
+    interface CipherOCBOptions extends stream.TransformOptions {
+        authTagLength: number;
```

Comment:
```suggestion
        authTagLength?: number | undefined;
```
can this be undefined at runtime? If so, please stick to conention here supporting TS4.4 strict optional property types

Reply:
OCB mode requires the options object and this property. I don't understand the suggestion.

Reply:
But i'll double check.

Reply:
> The options argument controls stream behavior and is optional except when a cipher in CCM or OCB mode (e.g. 'aes-128-ccm') is used. In that case, the authTagLength option is required and specifies the length of the authentication tag in bytes

The PR is correct as-is.

Reply:
in TS4.4 there is a new switch to control exact optional properties. Someone can use this in a setup, hence we are asking question about supporting this syntax feature (`prop?: type | undefined;`)

Reply:
Cool. Not applicable here though, the property cannot be undefined at runtime.

Reply:
thx for checking this out!